src/devices/Common/Iot/Device/Common/WeatherHelper.cs

```diff
@@ -0,0 +1,104 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Iot.Units;
+using System;
+
+namespace Iot.Device.Common
+{
+    /// <summary>
+    /// Helpers for weather. Formulas taken from http://www.reahvac.com/tools/humidity-formulas/.
+    /// </summary>
+    public class WeatherHelper
+    {
+        /// <summary>
+        /// The heat index (or apparent temperature) is used to measure the amount of discomfort
+        /// during the summer months when heat and humidity often combine to make it feel hotter
+        /// than it actually is. The heat index is usually used for afternoon high temperatures.
+        /// Use the summer simmer index instead for overnight low temperatures.
+        /// </summary>
+        /// <param name="airTemperature">The dry air temperature</param>
+        /// <param name="relativeHumidity">The relative humidity (RH) expressed as a percentage</param>
+        /// <returns>The heat index, also known as the apparent temperature</returns>
+        public static Temperature HeatIndex(Temperature airTemperature, double relativeHumidity)
```

Comment:
Since we talked about discoverability offline it might make sense to create a second class and use this there as an extension method, i.e.:

```csharp
public static  class WeatherExtensions
{
    public static Temperature HeatIndex(this Temperature airTemperature, double relativeHumidity)
        => WeatherHelpers.HeatIndex(airTemperature, relativeHumidity);
}
```

this should improve discoverability

Reply:
although you can keep this as is since I'm not sure it actually will improve it considering you have to use the namespace and therefore already know about it